faqs/galaxy/dataset_metadata_missing.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Missing or Incomplete Dataset metadata 
+area: datasets   
```

Comment:
I am not sure the title reflects what an end users would look for. I mean you would need to know that you have an issue with the metadata of your dataset.